tutorials/MaxSubstructualAlignment.ipynb

```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Molecule substructural alignment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are cases when we want to align molecules which only share \n",
+    "a fraction of their structure, like ligands in a congeneric series, where\n",
+    "a common scaffold or core is common to all ligands in the series.\n",
+    "We can perform this alignment using HTMD function: maximalSubstructureAlignment \n",
```

Comment:
Use backticks to highlight function/class/module names like this `maximalSubstructureAlignment`